src/components/BurgerMenu/BurgerMenu.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import './BurgerMenu.scss';
+
+export const CN = 'burger-menu';
+
+const BurgerMenu = ({ menuItemsList }) => {
+  const renderMenu = () => menuItemsList.map(({ itemName }) => {
+    return (
+      <Link key={itemName} to={'/404'} className={`${CN}__link`}>
```

Comment:
Do all links lead to page 404 ?

Reply:
At the moment all these pages do not exist, so all links lead to /404. When pages will be created, there will be a need to add proper paths as an array of objects and rewrite this a bit.